CRM/Utils/Token.php

```diff
@@ -1243,7 +1243,7 @@ public static function getTokenDetails(
     $numberofContacts = count($contactIDs);
     $query = new CRM_Contact_BAO_Query($params, $returnProperties);
 
-    $details = $query->apiQuery($params, $returnProperties, NULL, NULL, 0, $numberofContacts);
+    $details = $query->apiQuery($params, $returnProperties, NULL, NULL, 0, $numberofContacts, TRUE, FALSE, TRUE, CRM_Contact_BAO_Query::MODE_CONTACTS, NULL, TRUE);
```

Comment:
Whoooo, I'm scared of Jenkins. He always tells me things the hard way. But I agree and change it to your example call.